src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgWebHook.java

```diff
@@ -95,6 +95,7 @@ private static File getTrackingFile(String orgName) {
         return new File(Jenkins.get().getRootDir(), "github-webhooks/GitHubOrgHook." + orgName);
     }
 
+    // TODO never called?
```

Comment:
The uninstallation of an app sends out an `"action":"deleted"` installation event, similar to installation of an app which also sends out an installation event but with `"action":"created"`

Reply:
Yes but AFAICT this Java method is not called.

Reply:
Yeah looking at it, it might be designed to act on a deregister for a webhook, but I cannot find any such event for it on the Github API document :/

Reply:
I think the intention was for this to be called when the `OrganizationFolder` was deleted, but from https://github.com/jenkinsci/github-branch-source-plugin/commit/e424aa26c28789285ccd8fc5add40d4c6250f135 I am guessing this was never implemented.